oned/chunks.hpp

```diff
@@ -137,13 +137,22 @@ template <typename T> class Chunks {
     return reverse_iterator(begin());
   }
 
-  // Access the first chunk. Calling front on an empty Chunks results in
-  // undefined behavior.
-  constexpr Stripe<T> front() const { return (*this)[0]; }
+  class StripeProxy {
+  public:
+      StripeProxy(const Chunks *chunks, size_t index)
+          : chunks_(chunks), index_(index) {}
+
+      auto begin() const { return (*chunks_)[index_].begin(); }
+      auto end() const { return (*chunks_)[index_].end(); }
+      operator Stripe<T>() const { return (*chunks_)[index_]; }
+  private:
+      const Chunks *chunks_;
+      size_t index_;
+  };
+
+  constexpr StripeProxy front() const { return StripeProxy(this, 0); }
```

Comment:
All public methods should have doc comments. I think it is OK to just leave to old comments there:
```suggestion
  // Access the first chunk. Calling front on an empty Chunks results in
  // undefined behavior.
  constexpr StripeProxy front() const { return StripeProxy(this, 0); }
```